app/assets/stylesheets/overrides.scss

```diff
@@ -74,16 +126,44 @@ $font: $brown;
   padding: 15px;
 }
 
-table.center {
-  margin-left: auto;
-  margin-right: auto;
-  max-width: 500px;
-}
-
 .has-table {
   padding-bottom: 20px;
 }
 
 .example-table {
   font-style: italic;
+  align: center;
+}
+
+.exercise-table {
+  align: center;
+}
+
+.main-title a {
+  color: $titlecolor;
+}
+
+.button.round {
+  background-color: $navborder;
+  color: $navmainfont;
+}
+
+.full-width {
+  width: 100%;
+}
+
+#notice_wrapper {
+  background-color: $accent;
+}
+
+p.notice {
+  background-color: $accent;
+}
+
+.flash-alert {
+  background-color: $accent;
+}
+
+table {
+    border-spacing: 0 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces